Makefile

```
@@ -77,10 +77,6 @@ update-licenses: install-tools
 
 .PHONY: verify-licenses
 verify-licenses:
```

```suggestion
verify-licenses: install-tools
```

do you need this here too?

---

Yup